### DISCUSSION TITLE
Conf.file > undocumented params.

### DISCUSSION THREAD

**Discussion by Yuri-01** [AdguardTeam/AdGuardHome#3877] [Q&A]
Hi and thank you for your work.

Please explain, that means the following params in config file:
`os:
  group: ""
  user: ""
  rlimit_nofile: 0`

(не нашёл их описания в wiki (https://github.com/AdguardTeam/AdGuardHome/wiki/Configuration#configuration-file))

**Comment by ainar-g** [comment]
Ошибка документации, спасибо, что дали знать!  Исправим в #3881, как появится время.
